governance/votes/Executive vote - March 24, 2023.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Template - [Executive Vote] Emergency Spell for Shutdown Implications - March 24, 2023
+summary: Enables the Emergency Shutdown Module to remove Governance’s authority on the MCD_JOIN_CRVV1ETHSTETH_A, MCD_CROPPER, and CHANGELOG contracts to prevent malicious actions after shutdown.
+date: 2023-03-24T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Emergency Spell for Shutdown Implications - March 24, 2023
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Emergency Shutdown Module will be able to remove Governance’s authority on the MCD_JOIN_CRVV1ETHSTETH_A, MCD_CROPPER, and CHANGELOG contracts.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **16 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Emergency Shutdown Module Changes
+
+Based on this [recommendation](https://forum.makerdao.com/t/emergency-shutdown-governance-vulnerability-proposed-emergency-spell/20255) from the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7), the [Emergency Shutdown Module](https://manual.makerdao.com/governance/emergency-shutdown) will be able to remove Governance’s authority over the following contracts, if this executive proposal passes.
+
+#### Effected Contracts 
```

Comment:
```suggestion
#### Affected Contracts 
```

Reply:
I had it like this but retro convinced me otherwise, state your case ser

Reply:
Pablo is a grammar king and agrees, will stop listening to retro lol (only for grammar <3)